### DISCUSSION TITLE
Ignore composer tags in "all artists" view

### DISCUSSION THREAD

**Discussion by jeckman** [LMS-Community/slimserver#901] [General]
I understand lots of folks with classical collections rely heavily on composer tags - I generally ignore them. 

Challenge is those composers get included in the "all artists" views - is it possible to tell "all artists" (I use the separate "album artists" and "all artists" lists) to NOT include the composer tags? 

Feels like I have seen a setting somewhere but could not find one, and searching mostly turns up people talking about how to use those tags not how to exclude them. 

I could go through and blank the tags in my whole collection but would prefer not to

**Comment by michaelherger** [comment]
The option would be there if you were using the single artists list option. You can then define what you consider an artist worth being listed. "All artists" does what it says: it shows members of all known artist roles.

**Reply by jeckman** [reply]
Ah, that makes sense - I saw people referring to that setting but did not see it. 

I still think it would be valuable to be able to define what shows in All Artists even if you are separately using Album Artists, but I guess that becomes a feature request ;) 

thanks

**Reply by jeckman** [reply]
I suppose the "use single, configurable list of artists" in the selection on that screen should have been the clue

**Comment by jeckman** [comment]
Actually I was able to get what i'm looking for from this more flexible version thanks
